src/tmx/Messages/include/RSUStatusMessage.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+
+#include <tmx/messages/message.hpp>
+#include "MessageTypes.h"
+
+
+namespace tmx::messages {
+
+
+class RSUStatusMessage : public tmx::message
+{
+	public:
+		RSUStatusMessage() {}
```

Comment:
Does this message not have any attributes?

Reply:
No, we just need the content which is from tmx::message.